### DISCUSSION TITLE
Option to Display Comment/Revision Count

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7928] [Feature Requests]
I'd like to surface some system data as options for field displays. Specifically, I'd like the ability to show:

* Comment Count
* Revision Count

Each should have a nice Display option (eg: a comment icon followed by the count).

**Comment by rijkvanzanten** [comment]
(Another nice one for a built-in Insights dashboard maybe? 🤔 )

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
